apps/nuxt/src/tools/project/projectPriority.ts

```diff
@@ -0,0 +1,43 @@
+import { ProjectType } from '@/types'
+import Navigation from '@/tools/navigation'
+import { useFiltersStore } from '@/stores/filters'
+
+export default class ProjectPriority {
+  static getPriorityProjects(projects: ProjectType[] = []): ProjectType[] | undefined {
+    const filtersStore = useFiltersStore()
+
+    const hasThemeSelected = filtersStore.hasThemeTypeSelected()
+    const isCompanyDataSelected = filtersStore.getCompanyDataSelected().value
+    const isHomePage = new Navigation().isHomepage()
+
+    if (!isCompanyDataSelected || (isHomePage && !hasThemeSelected)) {
+      return undefined
+    }
+
+    const projectQty = hasThemeSelected ? 1 : 3
+    return projects.slice(0, projectQty)
+  }
+
+  static isPriorityProject(project: ProjectType, priorityProjects: ProjectType[] = []): boolean {
```

Comment:
suggestion: comme la classe s'appel déjà `ProjectPriority`, il y a peut être la possibilité de simplifier les noms des fonctions

- `get`
- `is`
- `getIndex`
- `isUnique`
- `has`
- ...